app/views/layouts/application.html.slim

```diff
@@ -28,4 +28,12 @@ html
     == render 'layouts/icons'
 
     javascript:
-      var SALESFORCE_BASE_URL = "#{{current_user&.admin ? ENV['SALESFORCE_INSTANCE_URL'] : ENV['COMMUNITY_LOGIN_URL']}}"
\ No newline at end of file
+      var SALESFORCE_BASE_URL = "#{{current_user&.admin ? ENV['SALESFORCE_INSTANCE_URL'] : ENV['COMMUNITY_LOGIN_URL']}}"
+
+    - if ENV['GOOGLE_ANALYTICS_TRACKING_ID']
+      script{ async src="https://www.googletagmanager.com/gtag/js?id=#{ENV['GOOGLE_ANALYTICS_TRACKING_ID']}" }
+      javascript:
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){ dataLayer.push(arguments); }
+        gtag('js', new Date());
+        gtag('config', "#{ENV['GOOGLE_ANALYTICS_TRACKING_ID']}")
```

Comment:
The main change for getting more events was removing the document event listener for `turbolinks:loaded` here